packages/client/hmi-client/src/components/Sidebar.vue

```diff
@@ -2,20 +2,30 @@
 /**
  * Sidebar component for navigating modes
  * */
-import { ref, computed } from 'vue';
-import Button from '@/components/Button.vue';
+import { ref } from 'vue';
+import IconOpenPanelRight16 from '@carbon/icons-vue/es/open-panel--filled--right/16';
+import IconSidePanelClose16 from '@carbon/icons-vue/es/side-panel--close/16';
 import IconDataPlayer32 from '@carbon/icons-vue/es/data-player/32';
-import DocumentPdf32 from '@carbon/icons-vue/es/document--pdf/32';
+import IconDocumentPdf32 from '@carbon/icons-vue/es/document--pdf/32';
 import IconMachineLearningModel32 from '@carbon/icons-vue/es/machine-learning-model/32';
 import IconTableSplit32 from '@carbon/icons-vue/es/table--split/32';
 import IconProvenanceGraph32 from '@carbon/icons-vue/es/flow/32';
```

Comment:
Maybe rename this to `IconFlow32` to match with the icon name? I think I did this naming haha.